torch/testing/_internal/fake_config_module.py

```diff
@@ -25,6 +25,9 @@
 e_env_default = Config(env_name_default="ENV_TRUE", default=False)
 e_env_default_FALSE = Config(env_name_default="ENV_FALSE", default=True)
 e_env_force = Config(env_name_force="ENV_TRUE", default=False)
+e_aliased_bool = Config(
+    alias="torch.testing._internal.fake_config_module2.e_aliasing_bool", default=True
```

Comment:
Should we allow defaults on aliases? (My assumption is that we should not allow any thing else if we allow an alias setting in a config). Since nothing else will actually apply when setting.

Reply:
hmm, what if the alias is not set? do we read the alias' default?

Reply:
Yep - that's what happens in your current code. There is really no concept of a "unset" config. It's just a series of rules to return a value. getattr will always return something, and we should always return that to the user.

(Note that in some cases, we need to return null, which is why we use _UNSETS_SENTINEL).

Reply:
Yeah, don't allow defaults on aliases.